types/adapters.d.ts

```diff
@@ -1,6 +1,14 @@
-import { InternalOptions } from "./internals"
-import { User, Profile, Session } from "."
+import { User, Session } from "."
 import { EmailConfig } from "./providers"
+import { Awaitable } from "./internals/utils"
+
+export interface AdapterUser extends User {
+  emailVerified?: Date | null
```

Comment:
`emailVerified` is only used internally, so the default `User` interface never exposes it.